js/authentication.md

```diff
@@ -561,6 +561,115 @@ export default class App extends React.Component {
 }
 ```
 
+#### Federated with Auth0
+
+You can use `Auth0` as one of the provider of your Cognito Federated Identity Pool so that you can allow your Auth0 users to have access to your AWS resources. 
+
+Step 1. Learn [how to integrate Auth0 with Cognito Federated Identity Pool](https://auth0.com/docs/integrations/integrating-auth0-amazon-cognito-mobile-apps)
```

Comment:
how to integrate Auth0 with Cognito Federated Identity Pools

Reply:
Done